crmsh/ui_resource.py

```diff
@@ -515,7 +516,7 @@ def ms_to_sec(m):
             import re
             failcount_res = re.findall(r'fail-count-{}#(.*)_([0-9]+)'.format(rsc), out)
             if not failcount_res:
-                context.fatal_error("No failcount on node {} for resource {}".format(node, rsc))
+                return
```

Comment:
Strictly,  it probably should return success here only if the specified `value` is `0`.

Reply:
I think it's fine to return success here, since no failcount here is unharmful

Reply:
Like said, it's not really an usual case that users manipulate failcounts and set them to non-zero value, but if crmsh happens to receive such a request and is refusing to set it to non-zero, it as least should tell users they are not getting the expected result .

Reply:
Thanks for pointing this @gao-yan ! Already changed as `return True if value and int(value) == 0 else False`